### ISSUE TITLE
pubDate missed for items in RSS091 parser

### ISSUE THREAD

**Issue by JampireX** [rometools/rome#414]
RSS 0.91 have **pubDate** tag for **items** but lib not parsing this tag in RSS091UserlandParser

RSS091UserlandParser
```
protected Item parseItem(Element rssRoot, Element eItem, Locale locale) {
        Item item = super.parseItem(rssRoot, eItem, locale);
        Element description = eItem.getChild("description", this.getRSSNamespace());
        if (description != null) {
            item.setDescription(this.parseItemDescription(rssRoot, description));
        }

        Element encoded = eItem.getChild("encoded", this.getContentNamespace());
        if (encoded != null) {
            Content content = new Content();
            content.setType("html");
            content.setValue(encoded.getText());
            item.setContent(content);
        }

        return item;
    }
```

Can be reproduced with next feed:
```
<rss version="0.91">
<channel>
    <title></title>
    <description></description>
    <link></link>
    <lastBuildDate></lastBuildDate>
    <language></language>
    <item>
        <title></title>
        <description></description>
        <pubDate>Fri, 15 Feb 2019 08:34:29 +0000</pubDate>
        <link></link>
    </item>
</channel>
</rss>
```

**Comment by mishako** [comment]
Hi and thank you for your contribution! Do you have a source for the "RSS 0.91 have pubDate tag for items" statement? I checked RSS 0.91 spec [1] and corresponding Userland spec [2] and couldn't find `pubDate` on item level.

[1] http://www.rssboard.org/rss-0-9-1-netscape
[2] http://backend.userland.com/rss091

**Comment by JampireX** [comment]
I'm sorry, i didn't check the specification for this version. I just found this parameter in the feed and couldn't take it using this library.
In this case it's probably just a specification violation but it would be good to support backward compatibility or methods for solving such cases in the future.

**Comment by mishako** [comment]
I agree with you, we should just take all available values regardless of the RSS version. I added some comments to your pull request.

**Comment by mishako** [comment]
Fixed in #415
